lib/ansible/modules/network/f5/bigip_firewall_rule.py

```diff
@@ -205,6 +205,7 @@
       - This parameter is mutually exclusive with many of the other individual-rule
         specific settings. This includes C(logging), C(action), C(source),
         C(destination), C(irule'), C(protocol) and C(logging).
+      - This parameter is only used when parent_policy is specified, otherwise it is ignored.
```

Comment:
suggest that you wrap the `parent_policy` word here with `C()` so that it reads `C(parent_policy)`. This will make it stand out as a code parameter in the documentation

Reply:
changes pushed and squashed @caphrim007